insights/client/phase/v1.py

```diff
@@ -131,6 +131,23 @@ def post_update(client, config):
         logger.debug('Entitling an AWS host. Bypassing registration check.')
         return
 
+    if config.show:
+        try:
+            client.show()
+            sys.exit(constants.sig_kill_ok)
+        except Exception as e:
+            print(e)
+            sys.exit(constants.sig_kill_bad)
+
+    if config.check_results:
+        try:
+            client.config.base_url += "/platform"
```

Comment:
Instead of concatenating `/platform` here, I would suggest using the `show` and `check_results` options to set legacy_upload = False in `InsightsConfig.imply_options`. That way it'll get set in auto_config.try_auto_configuration -- if someone were to set `legacy_upload=False` manually in the config, `/platform` would get added twice.